tests/config.bats

```diff
@@ -3,122 +3,78 @@
 load helpers
 
 @test "config-flags-order-verification" {
-  run buildah config cnt1 --author=user1
+  run_buildah 1 config cnt1 --author=user1
   check_options_flag_err "--author=user1"
 
-  run buildah config cnt1 --arch x86_54
+  run_buildah 1 config cnt1 --arch x86_54
   check_options_flag_err "--arch"
 
-  run buildah config cnt1 --created-by buildahcli --cmd "/usr/bin/run.sh" --hostname "localhost1"
+  run_buildah 1 config cnt1 --created-by buildahcli --cmd "/usr/bin/run.sh" --hostname "localhost1"
   check_options_flag_err "--created-by"
 
-  run buildah config cnt1 --annotation=service=cache
+  run_buildah 1 config cnt1 --annotation=service=cache
   check_options_flag_err "--annotation=service=cache"
 }
 
+function check_matrix() {
+  local setting=$1
+  local expect=$2
+
+  for image in docker oci; do
+      for which in Docker OCIv1; do
+        run_buildah --debug=false inspect --type=image --format "{{.$which.$setting}}" scratch-image-$image
+        is "$output" "$expect" "scratch-image-$image > .$which.$setting"
+    done
+  done
+}
+
```

Comment:
neat little function, made me think for a bit @edsantiago

Reply:
...which, to me, means: "needs a comment!". Will do; something like "matrix check all combinations of settings in each container". Thanks.